### PR TITLE
README.rst: coala banner too big on pypi page

### PR DESCRIPTION
**Pull request by RohanVB**
The resolution of the coala banner is
1920 x 693, it has now been resized to
1086 × 387 to fit the page and stop the
scrolling caused by the large image.

Fixes: https://github.com/coala/coala/issues/3340

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by RohanVB** [comment]
@Mixih Yeah, I was just fixing the table. I think maybe this would be a fix though? I checked the pypi site on mobile and it looks like the image is inline with the text, and since its inline on my laptop it'd render correctly on a lower res screen?

**Comment by Mixih** [comment]
Maybe... Anyways... I'm not going to ack until someone who understand pypi's formatting rules takes a look(they're really weird and strict) xD

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+.. image:: http://i.imgur.com/gOHfIWL.png
```
Comment (meetmangukiya) [inline README.rst]: I think you should create a pull at assets repo, and then use that URL, as used earlier.

Comment (meetmangukiya) [inline README.rst]: I am not sure of `imgur`'s expiration policy, thats why

Comment (RohanVB) [inline README.rst]: > Imgur keep images for ever as long as they are receiving at least 1 view every six months. If they are not they may be deleted to save space. Popular images and those shared on Imgur will probably meet this criteria due to becoming viral and being better accessible.

But I'll do the pull at assets repo.

**Comment by jayvdb** [comment]
ack 7299819

**Comment by Mixih** [comment]
rebase please

**Comment by Mixih** [comment]
reack b0d06dd

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**